cli/stream_command.go

```diff
@@ -2282,6 +2384,12 @@ func (c *streamCmd) askSource(name string, prefix string) *api.StreamSource {
 	}, &cfg.FilterSubject)
 	fisk.FatalIfError(err, "could not request filter")
 
+	err = askOne(&survey.Input{
+		Message: fmt.Sprintf("%s Subject mapping transform", prefix),
+		Help:    "Map matching subjects according to this transform destination",
+	}, &cfg.SubjectTransformDest)
+	fisk.FatalIfError(err, "could not request subject mapping destination transform")
```

Comment:
Yes source already had SubjectFilter, so to introduce transform there it makes sense to re-use the source's subject filter as the source for the transform, but obviously can't rename it to something other because that would break compatibility.

I think the better option here would be to not prompt for the transform (i.e. the destination) if the user didn't specify a filter

Reply:
Done: don't prompt for destination transform if filter subject is ""